input/fsh/profiles.fsh

```diff
@@ -304,7 +304,7 @@ Title: "Mapping from CGM Temporary Codes to LOINC"
 Description: "Mapping concepts from the CGM Summary code system to LOINC codes."
 * name  = "CGMSummaryToLoinc"
 * experimental = false
-* status = #draft
+* status = #active
```

Comment:
Should we just remove this row, and let it be controlled by the `sushi-config.yaml` file?

Reply:
either way works for me